### PR TITLE
add loguru, remove all std::cout/std::cerr replace by logging

### PR DESCRIPTION
**Pull request by Milerius**

### CODE REVIEW

File: service/service.hpp [hunk 1]
```diff
@@ -154,57 +174,58 @@ namespace raven
...
     }
 
     void update_setting(json::json &json_data, uvw::PipeHandle &sock)
     {
+        LOG_SCOPE_F(INFO, __PRETTY_FUNCTION__);
         auto cfg = fill_request<setting_update>(json_data);
-        std::cout << "cfg.id: " << cfg.id.value() << std::endl;
-        std::cout << "settings_to_update: " << cfg.settings_to_update.dump() << std::endl;
+        DLOG_F(INFO, "cfg.id: %d", cfg.id.value());
+        DLOG_F(INFO, "settings_to_update: %s", cfg.settings_to_update.dump().c_str());
```
Comment (Gaspard--) [inline service/service.hpp]: I think this is not correct as per what we discussed a few PRs ago.

File: service/db.hpp [hunk 2]
```diff
@@ -79,31 +83,34 @@ namespace raven
                 data_to_bind[config_name_keyword] = config_create_data[config_name_keyword];
                 data_to_bind["SETTINGS"] = json::json::object();
                 data_to_bind["OTHER_CONFIG"] = json::json::array();
-                std::cout << data_to_bind.dump() << std::endl;
+                DLOG_F(INFO, "json to insert in db: %s", data_to_bind.dump().c_str());
```
Comment (Gaspard--) [inline service/db.hpp]: See above comment.

File: service/db.hpp [hunk 3]
```diff
@@ -121,53 +129,61 @@ namespace raven
...
     {
+        LOG_SCOPE_F(INFO, __PRETTY_FUNCTION__);
         try {
             auto functor_receive_data = [&setting_update_data](const std::string json_text) {
                 auto json_data = json::json::parse(json_text);
                 for (auto&[key, value] : setting_update_data.settings_to_update.items()) {
                     json_data["SETTINGS"][key] = value;
                 }
-                std::cout << json_data.dump() << std::endl;
+                DLOG_F(INFO, "json after update: %s", json_data.dump().c_str());
```
Comment (Gaspard--) [inline service/db.hpp]: See above comment

File: service/service.hpp [hunk 4]
```diff
@@ -129,9 +148,10 @@ namespace raven
     template <typename Request>
     Request fill_request(json::json &json_data)
     {
+        LOG_SCOPE_F(INFO, __PRETTY_FUNCTION__);
         Request request;
         from_json(json_data, request);
-        std::cout << "json receive:\n" << std::setw(4) << json_data << std::endl;
+        DLOG_F(INFO, "json receive:%s", json_data.dump().c_str());
```
Comment (Gaspard--) [inline service/service.hpp]: See above comment

File: service/service.hpp (hunk 1 above)
Comment (Milerius) [reply service/service.hpp]: Temporary objects are destroyed as the last step in evaluating the full-expression ([intro.execution]) that (lexically) contains the point where they were created.

Comment (Gaspard--) [reply service/service.hpp]: Ok, solid response!

**Pull request merged**